db/schema.rb

```diff
@@ -427,17 +426,6 @@
     t.index ["topic_id"], name: "index_posts_on_topic_id"
   end
 
-  create_table "prospective_applicants", id: :serial, force: :cascade do |t|
-    t.string "name"
-    t.string "email"
-    t.string "phone"
-    t.integer "college_id"
```

Comment:
Can we remove `College` and `University` tables

Reply:
These are tracked in other cleanup issues.